src/validation.cpp

```diff
@@ -12,6 +12,7 @@
 #include "consensus/consensus.h"
 #include "consensus/merkle.h"
 #include "consensus/validation.h"
+#include "cuckoocache.h"
```

Comment:
Is using the public interface of CuckooCache::cache really using its "internals"?